aws/solutions/AmazonCloudWatchAgent/inline/amazon_linux.template

```diff
@@ -48,24 +58,24 @@ Resources:
               content: !Sub |
                 {
                   "metrics": {
+                    "namespace": "${MetricNamespace}",
                     "append_dimensions": {
-                      "AutoScalingGroupName": "${!aws:AutoScalingGroupName}",
-                      "ImageId": "${!aws:ImageId}",
-                      "InstanceId": "${!aws:InstanceId}",
-                      "InstanceType": "${!aws:InstanceType}"
+                      "InstanceId": "${!aws:InstanceId}"
                     },
                     "metrics_collected": {
                       "mem": {
                         "measurement": [
                           "mem_used_percent"
-                        ]
+                        ],
+                        "metrics_collection_interval": 15
```

Comment:
I am opposed to these changes. This file is being co-opted for an integration test. https://github.com/aws/amazon-cloudwatch-agent-test/pull/223 should just include a CFT rather than taking a dependency on this repo. These are sample templates, not integration test templates.

Reply:
I thought the point of the testing was to add coverage of this template specifically.

Regardless, we should minimize the changes to this template.
And changing the interval is not absolutely necessary.